internal/version/version.go

```diff
@@ -20,6 +20,32 @@ import _ "embed" // We embed the Envoy version so that we can cache it in CI
 // Override this via "-X github.com/tetratelabs/getenvoy/internal/version.GetEnvoy=XXX"
 var GetEnvoy = "dev"
 
-// Envoy is the default version to download. This is embedded for re-use in build and CI scripts.
-//go:embed envoy.txt
-var Envoy string
+// LastKnownEnvoy is the last known Envoy version, used to ensure help statements aren't out-of-date.
```

Comment:
hopefully, these docs better explain the compile vs runtime concern

Reply:
+1 thanks!